tests/iscsi/iscsi_server.pm

```diff
@@ -54,10 +55,20 @@ sub run {
     wait_still_screen(2, 10);
     type_string '132';
     wait_still_screen(2, 10);
-    send_key 'alt-u';                                                                   # un-check use authentication
+    if (is_sle('>=15')) {
+        send_key 'alt-l';                                                               # un-check bind all IPs
+    }
+    else {
+        send_key 'alt-u';                                                               # un-check use authentication
+    }
     wait_still_screen(2, 10);
     send_key 'alt-a';                                                                   # add LUN
-    send_key_until_needlematch 'iscsi-target-LUN-path-selected', 'alt-p', 5, 5;         # send alt-p until LUN path is selected
+    if (is_sle('>=15')) {
+        send_key_until_needlematch 'iscsi-target-LUN-path-selected', 'alt-l', 5, 5;     # send alt-l until LUN path is selected
```

Comment:
and then you can just reuse this variable here

Reply:
The two keyboard shortcuts are the completely unrelated. Even though they use the same keys, they do very different things.
(re)using thar var woud suggest a non-existant relation.